Tests/SourceKitBazelBSPTests/BazelTargetCompilerArgsExtractorTests.swift

```diff
@@ -464,25 +280,64 @@ let expectedSwiftResult: [String] = [
 
 /// Expected result of processing the example input for the Obj-C file.
 let expectedObjCResult: [String] = [
-    "-x", "objective-c", "-target", "arm64-apple-ios17.0-simulator", "-D_FORTIFY_SOURCE=1", "-fstack-protector",
-    "-fcolor-diagnostics", "-Wall", "-Wthread-safety", "-Wself-assign", "-fno-omit-frame-pointer", "-g",
+    "-x",
+    "objective-c",
+    "/private/var/tmp/_bazel_user/hash123/execroot/__main__/bazel-out/darwin_arm64-opt-exec-ST-d57f47055a04/bin/external/apple_support+/crosstool/wrapped_clang",
```

Comment:
This seems like it shouldn't be here, probably because the new logic is not skipping the first few args like the previous one